version.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "7.1.3-build.{height}",
```

Comment:
You shouldn't modify the version.json file here.

Reply:
Not sure where that change came from, I'll remove it.

Reply:
Thanks, will make it easier to integrate this and migrate the DataGrid in the future. I'll try and look more into that once we ship our upcoming release.

Reply:
The change to version.json has been reverted